docs/source/concept_models/vivarium_swissre_lungcancer/concept_model.rst

```diff
@@ -49,32 +59,24 @@ Vivarium CSU Lung Cancer Simulation
 
 .. _1.1:
 
+The re-insurance company SwissRe is interested in the expected impact of increases in lung cancer screening coverage on the rates of detected lung cancer over the next 20 years among the insured population in select areas in China. 
```

Comment:
For some reason I like avoiding putting the names of CSU clients in our docs. can you remove?

Reply:
Done!